lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1005,6 +1094,27 @@ estimateTxSize skeleton =
         sizeOf_ChainCode  = 34
         sizeOf_Attributes = 45 -- NOTE: could be smaller by ~34 for Icarus
 
+    -- native_script =
+    --   [ script_pubkey      = (0, addr_keyhash)
+    --   // script_all        = (1, [ * native_script ])
+    --   // script_any        = (2, [ * native_script ])
+    --   // script_n_of_k     = (3, n: uint, [ * native_script ])
+    --   // invalid_before    = (4, uint)
+    --      ; Timelock validity intervals are half-open intervals [a, b).
+    --      ; This field specifies the left (included) endpoint a.
+    --   // invalid_hereafter = (5, uint)
+    --      ; Timelock validity intervals are half-open intervals [a, b).
+    --      ; This field specifies the right (excluded) endpoint b.
+    --   ]
+    sizeOf_NativeScript script =
+      case script of
+        (RequireSignatureOf _) -> sizeOf_SmallUInt + sizeOf_Hash28
+        (RequireAllOf ss)      -> sizeOf_SmallUInt + sizeOf_Array + sumVia sizeOf_NativeScript ss
+        (RequireAnyOf ss)      -> sizeOf_SmallUInt + sizeOf_Array + sumVia sizeOf_NativeScript ss
+        (RequireSomeOf _ ss)   -> sizeOf_SmallUInt + sizeOf_UInt + sizeOf_Array + sumVia sizeOf_NativeScript ss
+        (ActiveFromSlot _)     -> sizeOf_SmallUInt + sizeOf_UInt
+        (ActiveUntilSlot _)    -> sizeOf_SmallUInt + sizeOf_UInt
```

Comment:
Suggestion: put each case match on a separate line, and use a 4-char indent.

https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#avoid-variable-length-indentation
https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#limit-line-length-to-80-characters